misd/phone-number-bundle/1.3/config/packages/misd_phone_number.yaml

```diff
@@ -0,0 +1,6 @@
+# To persist libphonenumber\PhoneNumber objects, add the Misd\PhoneNumberBundle\Doctrine\DBAL\Types\PhoneNumberType mapping to your application's config.
+# This requires: doctrine/doctrine-bundle
```

Comment:
Good comment. Im fine with this. 

Do you have `doctrine/doctrine-bundle` in your composer.json's `suggest` section?

Reply:
Yep :)

```json
"suggest": {
        "doctrine/doctrine-bundle": "Add a DBAL mapping type",
        "jms/serializer-bundle": "Serialize/deserialize phone numbers using JSM library",
        "symfony/serializer": "Serialize/deserialize phone numbers using Symfony library",
        "symfony/form": "Add a data transformer",
        "symfony/templating": "Format phone numbers in templates",
        "symfony/twig-bundle": "Format phone numbers in Twig templates",
        "symfony/validator": "Add a validation constraint"
},
```